docs/mine/lotus/message-pool.md

```diff
@@ -0,0 +1,76 @@
+
+```sh
+# Will print the last BaseFee in attoFIL
+lotus chain head | xargs lotus chain getblock | jq -r .ParentBaseFee
+```
+
+Apart from the _BaseFee_, every message in the message pool has the following gas-related fields:
+
+- **_GasLimit_**: it is measured in gas units. It a hard limit on the amount of gas that a message's execution can consume. A message consumes gas for every fundamental operation it takes (see [prices](https://github.com/filecoin-project/lotus/blob/d678fe4bfa5b4c70bcebd46cdc38aafc452b42d1/chain/vm/gas.go#L87)). Messages can fail if they runs out of gas fails, at which point every modification to the state is reverted. Whether the execution is successful or not, the miner is given a reward calculated as _GasLimit _ GasPremium\*.
+- **_GasPremium_**: it is measured in attoFIL/gas units. It indicates how much a miner earns as their reward for including a message. A message typically earns its miner _GasLimit \* GasPremium_ attoFIL.
+- **_GasFeeCap_**: it is measured attoFIL/gas units. Its purpose is to establish a hard limit on the amount of FIL a message can cost its sender. A sender is guaranteed that a message will never cost them more than _GasLimit \* GasFeeCap_, not including any value that the message includes for its recipient. The total fee (in attoFIL/gas) for a message is _GasPremium + BaseFee_. Given that _GasPremium_ is set by the sender, _GasFeeCap_ essentially serves as a safeguard against a high, unexpected _BaseFee_.
+
+Note that if a message's _GasFeeCap_ is lower than the _BaseFee_, then the remainder comes from the miner (as a penalty). If _BaseFee + GasPremium_ is less than the message's _GasFeeCap_, then the miner might not earn the entire _GasLimit \* GasPremium_ reward.
+
+## Checking for pending messages in the local pool
+
+If messages are not deemed attractive enough by miners to be included in new blocks, they may become stuck in the local message pool. This is usually a consequence of the _GasFeeCap_ being too low, for example, when the Network's _BaseFee_ is high. It can also be a consequence of the _GasPremium_ being too low if the network is congested.
+
+You can check for messages currently in the local pool with:
+
+```sh
+lotus mpool pending --local
+```
+
+For each message you will be able to see key information like the _GasLimit_, the _GasFeeCap_ and the _GasPremium_ values, explained above.
+
+## Replacing messages in the messages pool
+
+You can replace messages in the pool by pushing a new message with the same `Nonce` that has a **new `GasPremium` that is at least 25% larger than that of the original message**. To this effect, the easiest is to use:
+
```

Comment:
Can someone thumbs up that this info is correct? I have tracked the inner workings of the `--auto` flag and I don't quite see it taking this into account in the new calculation.

cc. @whyrusleeping